[2024-05-30T07:46:31.614+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-30T07:46:31.629+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: example_complex.create_entry_group_result2 manual__2024-05-30T07:46:29.496104+00:00 [queued]>
[2024-05-30T07:46:31.633+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: example_complex.create_entry_group_result2 manual__2024-05-30T07:46:29.496104+00:00 [queued]>
[2024-05-30T07:46:31.633+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-05-30T07:46:31.640+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): create_entry_group_result2> on 2024-05-30 07:46:29.496104+00:00
[2024-05-30T07:46:31.648+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=3567) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-05-30T07:46:31.650+0000] {standard_task_runner.py:63} INFO - Started process 3576 to run task
[2024-05-30T07:46:31.652+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'example_complex', 'create_entry_group_result2', 'manual__2024-05-30T07:46:29.496104+00:00', '--job-id', '6', '--raw', '--subdir', '/home/***/.local/lib/python3.12/site-packages/***/example_dags/example_complex.py', '--cfg-path', '/tmp/tmpypvz7a56']
[2024-05-30T07:46:31.659+0000] {standard_task_runner.py:91} INFO - Job 6: Subtask create_entry_group_result2
[2024-05-30T07:46:31.697+0000] {task_command.py:426} INFO - Running <TaskInstance: example_complex.create_entry_group_result2 manual__2024-05-30T07:46:29.496104+00:00 [running]> on host fc2d9abb473f
[2024-05-30T07:46:31.747+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='example_complex' AIRFLOW_CTX_TASK_ID='create_entry_group_result2' AIRFLOW_CTX_EXECUTION_DATE='2024-05-30T07:46:29.496104+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-30T07:46:29.496104+00:00'
[2024-05-30T07:46:31.751+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-30T07:46:31.752+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-05-30T07:46:31.752+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo create_entry_group_result2']
[2024-05-30T07:46:31.761+0000] {subprocess.py:86} INFO - Output:
[2024-05-30T07:46:31.763+0000] {subprocess.py:93} INFO - create_entry_group_result2
[2024-05-30T07:46:31.763+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-05-30T07:46:31.764+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-30T07:46:31.780+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=example_complex, task_id=create_entry_group_result2, run_id=manual__2024-05-30T07:46:29.496104+00:00, execution_date=20240530T074629, start_date=20240530T074631, end_date=20240530T074631
[2024-05-30T07:46:31.825+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-30T07:46:31.833+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-30T07:46:31.833+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
